tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -648,6 +649,15 @@ private static void checkSuboptions(UOption givenOptions, UnicodeSet allowedOpti
                             + CldrUtility.LINE_SEPARATOR
                             + "Use -? for help.");
         }
+        if (i == FIX) {
+            final UnicodeSet allowedFilters = new UnicodeSet().add('P').add('Q').add('V');
+            for (char c : givenOptions.value.toCharArray()) {
```

Comment:
bug here -- should check for null before calling toCharArray, see https://unicode-org.atlassian.net/browse/CLDR-17405

Reply:
See https://github.com/unicode-org/cldr/pull/3521